data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -344,7 +353,8 @@ t_result_with_sorted_manuscript_versions_array AS (
         result.senior_editor_details,
         result.author_names_csv,
         result.evaluations,
-        CONCAT(publication.publication_date, ' ', publication.utc_publication_time) AS rp_publication_timestamp
+        CONCAT(publication.publication_date, ' ', publication.utc_publication_time) AS rp_publication_timestamp,
```

Comment:
Not sure if I mentioned it in a previous PR. But here `rp_publication_timestamp` is not a timestamp but a string.
Either we name it `rp_publication_timestamp_str` and parse it in Python or parse it within BigQuery SQL.

The test data should match the type. But currently looks to be a parsed timestamp?